modules/management/kubectl-apply/README.md

```diff
@@ -0,0 +1,94 @@
+## Description
+
+This module simplifies applying Kubernetes manifests to GKE clusters. It provides flexible options for specifying manifests, allowing you to either directly embed them as strings content or reference them from URLs, files, templates, or entire files in directories.
+
+* **Manifest:**
+  * **Raw String:** Specify manifests directly within the module configuration using the `content: manifest_body` format.
+  * **File/Template/Directory Reference:** Set `source` to the path to:
+    * A single URL to a manifest file. Ex.: `https://github.com/.../myrepo/manfiest.yaml`.
+    * A single local YAML manifest file (`.yaml`). Ex.: `./manfiest.yaml`.
+    * A template file (`.tftpl`) to generate a manifest. Ex.: `./template.yaml.tftpl`. You can pass the variables to format the template file in `template_vars`.
+    * A directory containing multiple YAML or template files. Ex: `./manifests/`. You can pass the variables to format the template files in `template_vars`.
+
+### Example
+
+```yaml
+- id: existing-gke-cluster
+  source: modules/scheduler/pre-existing-gke-cluster
+  settings:
+    project_id: $(vars.project_id)
+    cluster_name: my-gke-cluster
+    region: us-central1
+
+- id: kubectl-apply
+  source: modules/management/kubectl-apply
+  use: [existing-gke-cluster]
+  settings:
+    - content: "A YAML body"
```

Comment:
Can we give real examples here ? 

For example, use 

* `Content` to create a namespace ? 
* `source` to a valid URL , like network object installation
* `template_vars` to have a real map and values ?